app/models/in_person_enrollment.rb

```diff
@@ -60,17 +60,25 @@ def self.needs_late_email_reminder(early_benchmark, late_benchmark)
   def self.needs_usps_status_check(check_interval)
     where(status: :pending).
       and(
-        where(status_check_attempted_at: check_interval).
-        or(where(status_check_attempted_at: nil)),
+        where(last_batch_claim_at: check_interval).
```

Comment:
is `check_interval` a range or a specific time stamp?

Reply:
This can be either, but it's typically been a range.

Reply:
check interval with a certain amount of time. The check_interval will need to be changed to something longer than the duration of the job, so it doesn't catch enrollments that have been claimed earlier. Prior to this, the status_check_attempted_at was used, it wasn't set until the enrollment actually processed. So, this could cause an enrollment to be in two batches if one was running long.